includes/Checker/Abstract_Check_Runner.php

```diff
@@ -413,6 +413,10 @@ final public function get_checks_to_run() {
 			->include( $check_slugs ) // Ensures only the checks with the given slugs are included.
 			->exclude( $excluded_checks ); // Exclude provided checks from list.
 
+		if ( ! empty( $excluded_checks ) ) {
```

Comment:
Why we check additionally for exclude checks? We already checked above 🤔

Reply:
In $check_slugs, there are all given slugs. After excluding, `$collection` will have list of checks (after excluding). But in next step we are again validating the checks with `$collection->require( $check_slugs )`. So error was triggered from there.

Reply:
I've raised PR #387, utilizing a different approach. Please review it when you have a moment.